nestjs/src/courses/team-distribution/dto/join-team.dto.ts

```diff
@@ -12,8 +12,12 @@ export class JoinTeamDto {
 export class JoinTeamDtoRes {
```

Comment:
Res - is "response", "result" or something else?

Reply:
fixed